command/namespace_patch.go

```diff
@@ -119,12 +120,12 @@ func (c *NamespacePatchCommand) Run(args []string) int {
 
 	secret, err := client.Logical().JSONMergePatch(context.Background(), "sys/namespaces/"+namespacePath, data)
 	if err != nil {
-		c.UI.Error(fmt.Sprintf("Error patching namespace: %s", err))
-		return 2
-	}
+		if re, ok := err.(*api.ResponseError); ok && re.StatusCode == http.StatusNotFound {
```

Comment:
How do you feel about a switch/case or if/else here with 1 return? I have no preference, just offering an alternative approach in case you hadn't considered it. It _may_ be more readable, but that's of course subjective.

For reference, in case you didn't want to type try it out locally, which I totally get!
```go
if re, ok := err.(*api.ResponseError); ok {
  if re.StatusCode == http.StatusNotFound {
      c.UI.Error("Namespace not found")
  } else {
      c.UI.Error("Error patching namespace")
  }
    
    return 2
}
```

OR

```go
if re, ok := err.(*api.ResponseError); ok {
  switch re.StatusCode {
    case http.StatusNotFound:
      c.UI.Error("Namespace not found")
    default:
      c.UI.Error("Error patching namespace")
  }
    
  return 2
}
```

Reply:
Oh, I like the switch option. I'll go with that, thanks.

Reply:
Oh actually, I might keep it as is because I think I only care about it being a response error in the case of the 404. Otherwise we can just log the raw error.